<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T_graph</class>
 <widget class="QMainWindow" name="T_graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>威腾股份(温度曲线图)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Login.qrc">
    <normaloff>:/Login/Resources/Myicon.ico</normaloff>:/Login/Resources/Myicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="Login.qrc">:/Login/Resources/T_icon.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Adobe 仿宋 Std R</family>
                <pointsize>22</pointsize>
               </font>
              </property>
              <property name="text">
               <string>母线温度数据变化图</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>298</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Adobe 宋体 Std L</family>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>即时温度：</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>sensor1：</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>18</width>
                  <height>38</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="T_sensor1">
                <property name="font">
                 <font>
                  <family>Adobe Arabic</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>19</width>
                  <height>63</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>sensor2：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="T_sensor2">
                <property name="font">
                 <font>
                  <family>Adobe Arabic</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>18</width>
                  <height>63</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>sensor3：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="T_sensor3">
                <property name="font">
                 <font>
                  <family>Adobe Arabic</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>13</width>
                  <height>63</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>sensor4：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="T_sensor4">
                <property name="font">
                 <font>
                  <family>Adobe Arabic</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="T_customplot" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Login.qrc"/>
 </resources>
 <connections/>
</ui>
